hcn/hcnnetwork_test.go

```diff
@@ -66,3 +66,61 @@ func TestListNetwork(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestAddPolicyNetwork(t *testing.T) {
+
+	network, err := CreateTestOverlayNetwork()
+	if err != nil {
+		t.Error(err)
+	}
+
+	remoteSubnetRoutePolicy, err := HcnCreateRemoteSubnetRoute()
+	if err != nil {
+		t.Error(err)
```

Comment:
Try using t.Fatal(err) instead of t.Error(err)

I have another PR modifying this across https://github.com/Microsoft/hcsshim/pull/356

Reply:
Done.